tests/aws/services/events/test_events_schedule.py

```diff
@@ -0,0 +1,347 @@
+                snapshot.transform.key_value("ReceiptHandle"),
+                snapshot.transform.regex(target_id, "<target-id>"),
+                snapshot.transform.regex(rule_name, "<rule-name>"),
+                snapshot.transform.regex(queue_name, "<queue-name"),
+            ]
+        )
+        snapshot.match("messages-first", messages_first)
+
+        time.sleep(60)
+        messages_second = sqs_collect_messages(aws_client, queue_url, min_events=1, retries=3)
+        snapshot.match("messages-second", messages_second)
+
+        # check if the messages are 60 seconds apart
+        time_messages_first = events_time_string_to_timestamp(
+            json.loads(messages_first[0]["Body"])["time"]
+        )
+        time_messages_second = events_time_string_to_timestamp(
+            json.loads(messages_second[0]["Body"])["time"]
+        )
+        time_delta = time_messages_second - time_messages_first
+        assert time_delta == timedelta(seconds=60)
+
+    @markers.aws.validated
+    def tests_schedule_rate_custom_input_target_sqs(
+        self, create_sqs_events_target, events_put_rule, aws_client, snapshot
+    ):
+        queue_url, queue_arn = create_sqs_events_target()
+
+        bus_name = "default"
+        rule_name = f"test-rule-{short_uid()}"
+        events_put_rule(Name=rule_name, EventBusName=bus_name, ScheduleExpression="rate(1 minute)")
+
+        target_id = f"target-{short_uid()}"
+        aws_client.events.put_targets(
+            Rule=rule_name,
+            EventBusName=bus_name,
+            Targets=[
+                {
+                    "Id": target_id,
+                    "Arn": queue_arn,
+                    "Input": json.dumps({"custom-value": "somecustominput"}),
+                },
+            ],
+        )  # cleanup is handled by rule fixture
+
+        response = aws_client.events.list_targets_by_rule(Rule=rule_name)
+        snapshot.match("list-targets", response)
+
+        time.sleep(60)
+        messages_first = sqs_collect_messages(aws_client, queue_url, min_events=1, retries=3)
+
+        snapshot.add_transformers_list(
+            [
+                snapshot.transform.key_value("MD5OfBody"),
+                snapshot.transform.key_value("ReceiptHandle"),
+                snapshot.transform.regex(target_id, "<target-id>"),
+                snapshot.transform.regex(queue_arn, "<queue-arn>"),
+            ]
+        )
+        snapshot.match("messages", messages_first)
+
+    @markers.aws.needs_fixing
+    @markers.snapshot.skip_snapshot_verify(
+        paths=[
+            # tokens and IDs cannot be properly transformed
+            "$..eventId",
+            "$..uploadSequenceToken",
+            # FIXME: storedBytes should be implemented
+            "$..storedBytes",
+        ]
+    )
+    @pytest.mark.xfail(
+        reason="This test is flaky is CI, might be race conditions"  # FIXME: investigate and fix
+    )
+    def test_scheduled_rule_logs(
```

Comment:
this test needs to be completely reworked, due to the unknown time requirement this will take, it is being deferred to a later stage since the test case is covered by other tests mostly.